packages/screens/Settings/SettingsScreen.tsx

```diff
@@ -1,26 +1,294 @@
-import { useSelector } from "react-redux";
+import { useState } from "react";
+import {
+  View,
+  StyleSheet,
+  Switch,
+  useWindowDimensions,
+  Pressable,
+  Text,
+} from "react-native";
+import { TextInput } from "react-native-gesture-handler";
+// import { useSelector } from "react-redux";
 
+import chevronRightSVG from "../../../assets/icons/chevron-right.svg";
+import { BrandText } from "../../components/BrandText";
+import { SVG } from "../../components/SVG";
 import { ScreenContainer } from "../../components/ScreenContainer";
 import { PrimaryButton } from "../../components/buttons/PrimaryButton";
-import {
-  selectAreTestnetsEnabled,
-  setAreTestnetsEnabled,
-} from "../../store/slices/settings";
-import { useAppDispatch } from "../../store/store";
+import { TertiaryButton } from "../../components/buttons/TertiaryButton";
+import { SpacerColumn } from "../../components/spacer";
+// import { selectAreTestnetsEnabled } from "../../store/slices/settings";
+// import { useAppDispatch } from "../../store/store";
 import { ScreenFC } from "../../utils/navigation";
+import {
+  primaryColor,
+  secondaryColor,
+  neutralA3,
+  neutral17,
+  neutral55,
+  neutral00,
+  neutral33,
+} from "../../utils/style/colors";
+import {
+  fontSemibold12,
+  fontSemibold14,
+  fontSemibold20,
+} from "../../utils/style/fonts";
+import { layout } from "../../utils/style/layout";
+import { UserProfileModal } from "./components/UserProfileModal";
+
+const mobileWidth = 768;
+const padWidth = 1024;
 
 export const SettingsScreen: ScreenFC<"Settings"> = () => {
-  const testnetsEnabled = useSelector(selectAreTestnetsEnabled);
-  const dispatch = useAppDispatch();
+  const { width } = useWindowDimensions();
+
+  // const testnetsEnabled = useSelector(selectAreTestnetsEnabled);
+  // const dispatch = useAppDispatch();
+
+  const [displayTestnet, setDisplayTestnet] = useState<boolean>(true);
```

Comment:
We could put all these in a single object and in a single useState, not to repeat so much.